File: GUI_FoodCart_Tracker/GUI_FoodCart_Tracker/FirstViewController.m
//
//  FirstViewController.m
//  GUI_FoodCart_Tracker
//
//  Created by Clint Jellesed on 4/13/16.
//  Copyright © 2016 Clint Jellesed. All rights reserved.
//

#import "FirstViewController.h"

@interface FirstViewController ()
@property (weak, nonatomic) IBOutlet UILabel *Clock;

@end

@implementation FirstViewController

- (void)viewDidLoad {
    [super viewDidLoad];
    // Do any additional setup after loading the view, typically from a nib.
    //[self updateTime];
   }
-(void)updateTime
{
    NSDateFormatter *dateFormat = [[NSDateFormatter alloc]init];
    [dateFormat setDateFormat:@"hh-mm-ss"];
    _Clock.text = [dateFormat stringFromDate:[NSDate date]];
    
    
    //call update every second
    [self performSelector:@selector(updateTime) withObject:self afterDelay:1.0];
}


- (void)didReceiveMemoryWarning {
    [super didReceiveMemoryWarning];
    // Dispose of any resources that can be recreated.
}

@end

File: GroupProfile2/GroupProfile2/FirstViewController.swift
//
//  FirstViewController.swift
//  GroupProfile2
//
//  Created by Clint Jellesed on 4/27/16.
//  Copyright © 2016 Clint Jellesed. All rights reserved.
//

import UIKit
import AVFoundation

class FirstViewController: UIViewController, UITableViewDataSource, UITableViewDelegate {
    @IBOutlet weak var tableView: UITableView!
    
    func tableView(tableView: UITableView, numberOfRowsInSection section: Int) -> Int {
        return 3
    }
    func tableView(tableView: UITableView, cellForRowAtIndexPath indexPath: NSIndexPath) -> UITableViewCell {
        let cell = self.tableView.dequeueReusableCellWithIdentifier("cell", forIndexPath: indexPath) as! CustomCell
        
        //cell.photo.image = Image[indexPath.row]
        //cell.textView.text = names[indexPath.row]
        return cell
    }
    
    @IBAction func PausePlay(sender: UISegmentedControl) {
        if sender.selectedSegmentIndex == 0 {
            backgroundMusicPlayer.play()
        }
        
        else if sender.selectedSegmentIndex == 1 {
            backgroundMusicPlayer.pause()
        }
        
    }

    var backgroundMusicPlayer = AVAudioPlayer()
    
    override func viewDidLoad() {
        super.viewDidLoad()
        // Do any additional setup after loading the view, typically from a nib.
        
        let prefs = NSUserDefaults.standardUserDefaults()
        
        if prefs.boolForKey("ColorBlind") {
            self.view.backgroundColor = UIColor.whiteColor()
        }
        else {
            self.view.backgroundColor = UIColor.lightGrayColor()
        }
        //playSound("BGMusic2")
        //audioPlay.play()
        playBackgroundMusic("BGMusic2.mp3")

    }

    override func didReceiveMemoryWarning() {
        super.didReceiveMemoryWarning()
        // Dispose of any resources that can be recreated.
    }
    
    func playBackgroundMusic(filename: String) {
        let url = NSBundle.mainBundle().URLForResource(filename, withExtension: nil)
        guard let newURL = url else {
            print("Could not find file: \(filename)")
            return
        }
        do {
            backgroundMusicPlayer = try AVAudioPlayer(contentsOfURL: newURL)
            backgroundMusicPlayer.numberOfLoops = -1
            backgroundMusicPlayer.prepareToPlay()
            backgroundMusicPlayer.play()
        } catch let error as NSError {
            print(error.description)
        }
    }
    
}

